docs-src/concepts/what-is-a-workflow-execution.md

```diff
@@ -38,6 +38,16 @@ Scalability is responsiveness in the presence of load.
 A single Workflow Execution is limited in size and throughput but is scalable because it can [Continue-As-New](/concepts/what-is-continue-as-new) in response to load.
 A Temporal Application is scalable because the Temporal Platform is capable of supporting millions to billions of Workflow Executions executing concurrently, which is realized by the design and nature of the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) and [Worker Processes](/concepts/what-is-a-worker-process).
 
+### Replays
```

Comment:
This is a bit confusing wording.

The term "replay" in Temporal means a workflow execution replay, but how you're using "replays" here is specifically related to "replayers".

It is important to differentiate these two. Replays happen all the time in Temporal whether a user sees it or not. The part that you're referring to here is the "Replayer" which is a manually triggered replay I think.

If you are meaning the general concept of a "replay", then I don't think this is a good definition:

> A Replay is effectively a retry of the Workflow Execution that checks against an existing Event History.

Specifically, "retry" is a bad term here. Maybe "rerun" of "code". Also, that it checks against history is true but not necessarily the definition of a replay. Maybe there's some other way to make it clear that the effects of the code are not applied if those effects are already present in history and we check to make sure the effects match.